js/views/modals/Settings/Advanced/Appearance.js

```diff
@@ -0,0 +1,37 @@
+import app from '../../../../app';
+import loadTemplate from '../../../../utils/loadTemplate';
+import baseVw from '../../../baseVw';
+
+export default class extends baseVw {
+  constructor( options = { } ) {
+    super( options );
+
+    this.options = options;
+    this.settings = app.settings.clone( );
+  }
+
+  getFormData( ) {
+    return super.getFormData( this.$formFields );
+  }
+
+  render( ) {
+    this.settings = app.settings.clone( );
+    const mySettings = this.settings.toModelFormatJSON().appearanceSettings;
+
+    if ( mySettings.windowControlStyle == 'mac' )
+      app.localSettings.set( { 'macStyleWinControls' : true } );
+    else
+      app.localSettings.set( { 'macStyleWinControls' : false } );
```

Comment:
@rmisio agree that setting in the render was not something that works that well. I just wanted to get the functionality of the save working quickly to see it all come together, and didn't consider putting it somewhere else for that.